source/css/_common/components/third-party/disqusjs.styl

```diff
@@ -0,0 +1,39 @@
+if (hexo-config('disqusjs.enable') and hexo-config('darkmode')) {
+  @media (prefers-color-scheme:dark) {
+      html #dsqjs a {
```

Comment:
The indentation seems incorrect here, should be 2 spaces

Reply:
fixed